### DISCUSSION TITLE
Error when Texturing - received signal 22 (SIGABRT)

### DISCUSSION THREAD

**Discussion by Arkads-PL** [alicevision/Meshroom#1982] [Q&A]
Hello. I'm new to Meshroom and trying to do examples from YT

https://www.youtube.com/watch?v=ISAKM6sVtwA&list=PLqiB3IIUNAnUczpKbV1q7g3pHLRGmXNI_&index=1

I used images of the Whell but got the error using Meshroom 2023.1.0 in the last step Texturing.

Fragment of the Log below:
```
[21:49:37.299133][info] Load dense point cloud.
**[21:49:37.474171][warning] Alembic LOAD: NO OBSERVATIONS_WITH_FEATURES: , mvg_visibilityFeatId: 0, mvg_visibilityFeatPos: 0, OBSERVATIONS_WITH_FEATURES flag: false**
[21:49:38.158341][info] Found 1 image dimension(s): 
[21:49:38.158341][info] 	- [4272x2848]
[21:49:38.210202][info] Overall maximum dimension: [4272x2848]
[21:49:38.210202][info] Load input mesh.
[21:49:40.373976][info] Convert dense point cloud into ref mesh
[21:49:40.909579][info] remapMeshVisibility based on triangles start.
[21:49:41.852059][info] remapMeshVisibility done.
[21:49:41.865024][info] Input mesh has no UV coordinates, start unwrapping (LSCM)
[21:49:41.928853][info] Start mesh atlasing (using Geogram LSCM).
OpenNL assertion failed: param > 0, file:C:\dev\vcpkg-latest\buildtrees\geogram\src\5296ce80d3-9df1bba3f7.clean\src\lib\geogram\NL\nl_api.c, line:193
Abnormal program termination: received signal 22 (SIGABRT)
```

I don't have directory C:\dev\
How to fix it?

Best regards

Arek

**Comment by cbentejac** [comment]
Hi @Arkads-PL,

The log you're getting doesn't mean that Geogram cannot be found on your system (the path that is displayed is the one that was used when the release was built, and it is correctly included in the package), it just means that there an issue with mesh that's provided to the `Texturing` node. 

I don't know whether you specifically want to use the LSCM unwrapping or if you were just following the tutorial, but if you want to use LSCM unwrapping, you should probably play around with the `Meshing` node's options: using the default photogrammetry pipeline with the images provided in the tutorial and the LSCM unwrapping, I got the same error as you, and I was able to go through by changing some parameters of the `Meshing` node, which confirms that Geogram is correctly found.

If you're just trying to get results and do not specifically want to use LSCM unwrapping, the default photogrammetry pipeline (with the basic unwrapping method) works as well. 

I hope this helps!

**Comment by Arkads-PL** [comment]
Hi @cbentejac 

I tried, but it doesn't work. What should I change in the Meshing options? I have no error in the log now, but the Texturing is red and it's not done. Maybe it's a hardware problem?

Greeting

Arek
